### ISSUE TITLE
Accessibility: VoiceOver scrolling impossible in LazyList

### ISSUE THREAD

**Issue by ialokim** [JetBrains/compose-multiplatform#2271]
When using VoiceOver to navigate through a lazy list such as `LazyColumn`, only the visible items are read out (and additionally some not visible at the end of the list, but in a wrong order):

```kotlin
fun main() = singleWindowApplication {
    val state = rememberLazyListState()

    Box {
        LazyColumn(Modifier.fillMaxHeight(), state) {
            items(50) { x ->
                Text("item #$x")
            }
        }
        VerticalScrollbar(rememberScrollbarAdapter(state), Modifier.align(Alignment.CenterEnd).fillMaxHeight())
    }
}
```

I would expect VoiceOver to continue stepping through the list instead, as it does for a scrollable `Column`:

```kotlin
fun main() = singleWindowApplication {
    val state = rememberScrollState()

    Box {
        Column(Modifier.verticalScroll(rememberScrollState()).fillMaxSize()) {
            repeat(50) { x ->
                Text("item #$x")
            }
        }
        VerticalScrollbar(rememberScrollbarAdapter(state), Modifier.align(Alignment.CenterEnd).fillMaxHeight())
    }
}
```

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2271">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
